deployment/docker-compose/docker-compose.yml

```diff
@@ -18,7 +18,18 @@ services:
       watch:
         - action: rebuild
           path: ../../backend
-
-
+  chainlit-app:
+    build:
+      context: ../../chainlit
+      dockerfile: Dockerfile.chainlit
+    ports:
+      - "8100:8100"
+    volumes:
+      - .:/app  # Mount your project folder
```

Comment:
Also, this means that you will need the code base in order to run docker-compose. You won't be able to just download the docker images and run them.